json/src/number.rs

```diff
@@ -82,7 +82,7 @@ impl Number {
         }
     }
 
-    /// Converts a finite f64 to a Number. Infinite or NaN values are not JSON
+    /// Converts a finite `f64` to a number. Infinite or NaN values are not JSON
```

Comment:
Oops you had this one right before. This one really does refer to a `serde_json::Number`.